packages/shared/src/components/TagLinks.tsx

```diff
@@ -12,21 +13,24 @@ import { getTagPageLink } from '../lib/links';
 interface TagLinkProps {
   tag: string;
   className?: string;
+  isSelected?: boolean;
   buttonProps?: ButtonProps<'a'>;
 }
 
 export function TagLink({
   tag,
   className,
+  isSelected = false,
   buttonProps = {},
 }: TagLinkProps): ReactElement {
   return (
     <Link href={getTagPageLink(tag)} passHref key={tag} prefetch={false}>
       <Button
         tag="a"
         size={ButtonSize.XSmall}
-        variant={ButtonVariant.Float}
+        variant={isSelected ? ButtonVariant.Primary : ButtonVariant.Float}
```

Comment:
Hey. I understand this was the initial plan, but we've decided to make the selected tags look like the button below. There is a `TagElement` component that utilizes this design, I think a component called `AlertDot` is being rendered.

![Screenshot 2024-12-06 at 12 13 28 AM](https://github.com/user-attachments/assets/5a989628-8103-4d56-92f0-0ef893166506)

Reply:
Thanks @sshanzel,
To confirm, did you mean switching to `TagElement` instead of `TagLink`? (If so, that would significantly change the tag page design.)
Or should we stick with `TagLink` (existing design) and simply add the `AlertDot` to highlight?

Reply:
We should stick with `TagLink`, just add the `AlertDot` to highlight 🙏

Reply:
@nevilparmar11 I understand you have personal responsibilities too, hence I took the liberty to apply the changes. Hope you don't mind 🙏

Reply:
@sshanzel  No worries, appreciate you taking it on 🙌. I've been a bit busy this week.